graphql_api/query/admin_message.py

```diff
@@ -0,0 +1,24 @@
+import graphene
+
+from controller.auth import manager as auth
+from graphql_api.types import AdminMessage
+from controller.admin_message import manager
+
+
+class AdminMessageQuery(graphene.ObjectType):
+    all_admin_messages = graphene.Field(
```

Comment:
maybe add limit that defaults to 100 (though 100 will take some time until we reach it i'd use it to have a failsafe)

Reply:
done